content/en/docs/setup/install/istioctl/index.md

```diff
@@ -650,7 +650,15 @@ The --set flag also creates any intermediate nodes in the path that are missing
 
 ## Uninstall Istio
 
-To uninstall Istio, run the following command:
+To uninstall Istio completely, run the following command:
+
+{{< text bash >}}
+$ istioctl x uninstall <your original installation options> --purge
```

Comment:
Is it required to pass `<your original installation options>`?

I think just passing `istioctl x uninstall --purge` will remove all control planes and shared resources?

Reply:
yep it is not required actually, removed it